BannerlordTwitch/BLTAdoptAHero/Actions/AdoptAHero.cs

```diff
@@ -240,6 +247,33 @@ private static (bool success, string message) ExecuteInternal(string args, strin
                 return (false, "You can't adopt a hero: no available hero matching the requirements was found!");
             }
             
+            if (settings.SpawnInParty)
```

Comment:
Do you know what the implications are of adding a hero to party that isn't also a companion? It seems a bit weird, but I remember some quests do something like this so maybe its fine? But can the player *remove* them again? What happens if they do? This seems like something that needs some comprehensive testing!

Reply:
Alone, this command don't have a lot of interest. If this command is the only selected one, you will not be able to remove them from you party, you will have to search for them in cities to get them back and you will have to buy them again.

But, it's interesting with other commands. However, some may find it too strong to spawn heroes in party and will want to disable it.